AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -202,6 +204,12 @@ - (instancetype)initWithFrame:(CGRect)frame collectionViewLayout:(UICollectionVi
   // and should not trigger a relayout.
   _ignoreMaxSizeChange = CGSizeEqualToSize(_maxSizeForNodesConstrainedSize, CGSizeZero);
   
+  // Register the default layout inspector delegate for flow layouts, custom layouts
+  // will need to roll their own ASCollectionViewLayoutInspecting implementation.
+  _layoutDelegate = [self flowLayoutInspector];
```

Comment:
If you follow the trail of this the flow layout inspector returns nil if the layout is indeed not a flow layout. You bringing this up shows that this isn't super clear. Will clarify this with a conditional in the method body.